### DISCUSSION TITLE
Please reopen my tickets ("Sorry, this issue will be closed soon.")

### DISCUSSION THREAD

**Discussion by probonopd** [linuxdeepin/developer-center#2841] [Features Request & Ideas | 特性请求 & 头脑风暴]
- https://github.com/linuxdeepin/deepin-desktop-schemas/issues/2
- linuxdeepin/developer-center#2376
- https://github.com/linuxdeepin/qt5platform-plugins/issues/7
- https://github.com/linuxdeepin/developer-center/issues/767

were closed without comment reagrding the issue or the solution.

- Has it been looked at?
- Has it been worked on?
- Has it been discussed internally?
- Has it been fixed?
- Has it been declined?

It takes time and effort to create and submit tickets, so please at least include some information what happened to it when closing tickets.

> Sorry, this issue will be closed soon. If it is necessary to discuss it again, please create a new issue.

does not give any information. Yes it is "still necessary to discuss" because it was not discussed yet at all.

Thank you very much.

**Comment by lxwndr** [comment]
@tinali722 please, give special attention to this user!

**Comment by tinali722** [comment]
Noted Alex. On it now.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "linuxdeepin/developer-center"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月25日(星期天) 凌晨1:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [linuxdeepin/developer-center] Please reopen my tickets ("Sorry, this issue will be closed soon.") (#2080)





 
@tinali722 please, give special attention to this user!
